src/figuresClasses.ts

```diff
@@ -2,10 +1,90 @@
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public color: string,
+    private a: number,
+    private b: number,
+    private c: number,
+  ) {
+    if (a <= 0) {
+      throw new Error("Side 'a' must be greater than zero.");
+    }
+
+    if (b <= 0) {
+      throw new Error("Side 'b' must be greater than zero.");
+    }
+
+    if (c <= 0) {
+      throw new Error("Side 'c' must be greater than zero.");
+    }
+
+    const sides = [a, b, c].sort((x, y) => x - y);
```

Comment:
The array `sides` is created using `a`, `b`, and `c`, but these should be `this.a`, `this.b`, and `this.c` to refer to the class properties.